### DISCUSSION TITLE
[Icon Request] Yarn icon

### DISCUSSION THREAD

**Discussion by totkeks** [ryanoasis/nerd-fonts#993] [Ideas]
I'd like to request an additional icon from the devicons set: `yarn`, another node version manager, which is currently missing from the nerd font.

**Comment by Finii** [comment]
Thanks for the idea!

![image](https://user-images.githubusercontent.com/16012374/200553985-d552bbc6-ea3c-4583-a608-5bab7e467515.png)

What comes to my mind:

* Add one icon or update all devicions?
* Table of glyph source updates https://github.com/ryanoasis/nerd-fonts/issues/365#issuecomment-1254969225
* We could update `devicons` I guess, but there is the question of codepoint-stability.

**Reply by Finii** [reply]
Maybe the devicon update is something for `v3.0.0`, due to the codepoint stability issue :unamused: 
I still have no real idea there.

**Reply by totkeks** [reply]
Either is fine by me. Just noticed it was missing in my prompt and came here to ask.

If you add just the single icon now, that would be nice and having it with the next release with all the other devicons is great too.
